.github/workflows/deploy_aks.yml

```diff
@@ -0,0 +1,109 @@
+name: Build and deploy to AKS cluster
```

Comment:
can we change the wf name from deploy_aks.yml to build_and_deploy.yml or similar

Reply:
renamed file